datalab/datalab_session/data_operations/fits_output_handler.py

```diff
@@ -0,0 +1,35 @@
+import tempfile
+import numpy as np
+from astropy.io import fits
+
+from datalab.datalab_session.file_utils import create_jpgs
+from datalab.datalab_session.s3_utils import save_fits_and_thumbnails
+
+
+class FITSOutputHandler():
+    
+  def __init__(self, key: str, data: np.array, comment: str=None) -> None:
+      self.key = key
+      self.primary_hdu = fits.PrimaryHDU(header=fits.Header([('KEY', key)]))
```

Comment:
I think we can probably think of a more descriptive key then 'KEY' for the cache key. Like maybe DLABID or DLABKEY (since they are limited to 8 chars)

Reply:
you marked this as resolved but doesn't look like you changed it to anything?

Reply:
annnnnd he just left for Japan

Reply:
@LTDakin please slow down and make your way through the PR comments and not rush to merge once the first approval comes through. I'll start keeping things in "changes requested" for the time being.

Reply:
He'll fix it in prod!